### ISSUE TITLE
msg2eml: html displays internal msg styling

### ISSUE THREAD

**Issue by datsteves** [lolo101/MsgViewer#133]
Some MSGs when converting to EML do get styling information that seems like it would be the internal ones from how MSG itself encodes it.

This is how it is looking in the EML (from the test file)

```text
{\rtf1\ansi\ansicpg1252\fromtext \fbidis \deff0{\fonttbl

{\f0\fswiss\fcharset0 Arial;}

{\f1\fmodern Courier New;}

{\f2\fnil\fcharset2 Symbol;}

{\f3\fmodern\fcharset0 Courier New;}}

{\colortbl\red0\green0\blue0;\red0\green0\blue255;}

\uc1\pard\plain\deftab360 \f0\fs20 \par

\par

\par

Mit sonnigen Gr\'fc\'dfen\par

\par

Oliver Ermisch\par

\par
```

while the text version of it is just
```
Mit sonnigen Gr=C3=BC=C3=9Fen

Oliver Ermisch
```

for a workaround, I currently just check if `{\` is in the HTML version, and if so I just use the text version. which isn't ideal, but does work for the need that I have. So not an important issue, in my opinion, just something I noticed.

### Test File
[html-formatting.msg.zip](https://github.com/lolo101/MsgViewer/files/9583997/html-formatting.msg.zip)

**Comment by lolo101** [comment]
Hi @datsteves and thank you for the report !
Every issue is important :)
The RTF content in the original .msg message is written as HTML content to the target .eml message. That's definitely a bug.

**Comment by lolo101** [comment]
@datsteves please let me know of the fix doesn't work as intended. In particular with .msg with HTML encoded in RTF. Both HTML and RTF should be rendered when saved as .eml

**Comment by datsteves** [comment]
@lolo101 do I understand it right that both the RTF and the HTML Version should be included in the EML file for that test file in this ticket? if so, then the HTML Version is missing. but the RTF Version has now the right content type. Which already helps with the problem that I faced :)

**Comment by lolo101** [comment]
Sorry for the confusion.
You understood correctly what I said and what I said was what I intended to achieve but I realized afterward that I had a false assumption about the code: I thought the decompression of the RTF and the conversion of RTF to HTML occurred at the same time whereas it's not the case.
So indeed the issue is fixed, thank you very much for confirming that, but still, the code doesn't behave exactly as I expect it to.
So I opened #140  :smiley:
